pk_tic_tac_toe.rb

```diff
@@ -1,17 +1,28 @@
 class TicTacToeGame
+  VICTORY = [
+    [0, 1, 2], # same gp top row.
+    [3, 4, 5], # same gp middle row.
+    [6, 7, 8], # same gp bottom row.
+    [0, 3, 6], # same gp left column.
+    [1, 4, 7], # same gp center column.
+    [2, 5, 8], # same gp right column.
+    [0, 4, 8], # same gp left diagonal.
+    [6, 4, 2]  # same gp right diagnal.
+  ]
+
   attr_accessor :board
 
   def initialize
     @board = [*1..9]
-    @@remaining_spaces = 9
+    @remaining_spaces = 9
```

Comment:
I changed "remaining_space" from a class variable to an instance variable as it makes more sense to not associate this with the entire class but instead with instances.